TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/TeleOp.java

```diff
@@ -0,0 +1,238 @@
+package org.firstinspires.ftc.teamcode.teleop;
+
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+import com.qualcomm.robotcore.hardware.CRServo;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.Servo;
+import com.qualcomm.robotcore.util.Range;
+
+import org.firstinspires.ftc.teamcode.subsystems.hardware.Hardware;
+import org.firstinspires.ftc.teamcode.subsystems.hardware.HardwareElement;
+
+@com.qualcomm.robotcore.eventloop.opmode.TeleOp(name = "TeleOp", group = "OpMode")
+public class TeleOp extends OpMode {
+    private Hardware robot;
+    double leftFPower;
+    double rightFPower;
+    double leftBPower;
+    double rightBPower;
+    double drive;
+    double turn;
+    double strafe;
+    double intakePower = 0.5;
+    private final double armStartPos = 0.0;
+    private final double armTargetPos = 0.0;
+    private final double armTolerance = 2.5;
+    private final double viperSlideStartPos = 0.0;
+    private final double viperSlideTargetPos = 0.0;
+    private final double jointStartPos = 0.0;
+    private final double jointTargetPos = 0.0;
+
+    @Override
+    public void init() {
+        telemetry.addData("Status", "Initializing");
+
+        // Do all init stuff.
+        robot = new Hardware(hardwareMap, telemetry);
```

Comment:
tbc, this line isn't "doing all init stuff", it's creating a place to store all of our hardwareElements